site/specs/numbers.json

```diff
@@ -1420,8 +1420,9 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves bulk port-in orders for the specified accountId.\nA maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter is provided, the order results will be limited to the last two years.</p>\n",
-        "operationId": "GET /accounts/{accountId}/bulkPortins",
+        "description": "<p>Retrieves bulk port-in orders for the specified accountId.\nA maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter is provided, the order results will be limited to the last two years. <a href=\"https://dev.bandwidth.com/docs/numbers/guides/bulkportins\">Here is link to Guides and Tutorials</a></p>\n",
+        "operationId": "ListBulkPortins",
+        "summary": "List bulk port-in orders",
```

Comment:
@hamermike would it be best to update `operationId` and `summary` to more friendly names all at once, or is it ok if we have a mix?

Reply:
Either way is probably fine as long as we keep track. I'd say it might be easiest all at once. We can clean up the paragraph tags here as well. I think you can remove the newlines as well unless it looks out of place without them. I know the RAML used them quite a bit.